app/moves/steps.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
     controller: Assessment,
     next: 'save',
     heading: 'moves:steps.health_information.heading',
+    buttonText: 'actions:schedule_move',
```

Comment:
It's a little weird passing a key for the view to translate rather than the actual text.

Reply:
This file doesn't have access to the request object which contains the translation method though.

It's the way the [form wizard](https://github.com/UKHomeOffice/passports-form-wizard) actually handled it too. It's possibly something we could revisit later?

Reply:
This was an issue I encountered on #120 as well.`t` should be available outside of the views. I'm happy to leave it for now if we can add this to the issues list.